src/pamiq_core/interactions/interval_adjustors.py

```diff
@@ -0,0 +1,63 @@
+import math
+from abc import ABC, abstractmethod
+
+from pamiq_core import time
+
+
+class BaseIntervalAdjustor(ABC):
```

Comment:
[コーディング規約](https://github.com/MLShukai/pamiq-core/wiki/コーディング規約)より、基底クラスに`Base` prefixはつけないようになっている。
```suggestion
class IntervalAdjustor(ABC):
```